apps/api/src/app/controllers/resources/media-item.controller.ts

```diff
@@ -0,0 +1,37 @@
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { ApiOkResponse, ApiParam, ApiTags } from '@nestjs/swagger';
+import { MediaItemQueryService } from '../../../domain/services/media-item-query.service';
+import { resourceTypes } from '../../../domain/types/resourceTypes';
+import { isInternalError } from '../../../lib/errors/InternalError';
+import { isNotFound } from '../../../lib/types/not-found';
+import { MediaItemViewModel } from '../../../view-models/buildViewModelForResource/viewModels/media-item.view-model';
+import httpStatusCodes from '../../constants/httpStatusCodes';
+import { buildByIdApiParamMetadata, RESOURCES_ROUTE_PREFIX } from '../resourceViewModel.controller';
+import buildViewModelPathForResourceType from '../utilities/buildViewModelPathForResourceType';
+
+@ApiTags(RESOURCES_ROUTE_PREFIX)
+@Controller(
+    `${RESOURCES_ROUTE_PREFIX}/${buildViewModelPathForResourceType(resourceTypes.mediaItem)}`
```

Comment:
break out a helper for this

Reply:
On second thought, it's a bit premature. Let's wait until we do the next controller.